src/WebJobs.Script.WebHost/FunctionsSyncService.cs

```diff
@@ -42,8 +42,10 @@ internal bool ShouldSyncTriggers
         {
             get
             {
-                return _primaryHostStateProvider.IsPrimary &&
+                var shouldSyncTrigger = _primaryHostStateProvider.IsPrimary &&
                        (_scriptHostManager.State == ScriptHostState.Running);
+                _logger.LogDebug("ShouldSyncTriggers: '{shouldSyncTrigger}',PrimaryHostStateProvider.IsPrimary: '{primaryHostStateProvider.IsPrimary}',ScriptHostState: '{scriptHostManager.State}'", shouldSyncTrigger, _primaryHostStateProvider.IsPrimary, _scriptHostManager.State);
```

Comment:
Should we use LoggerMessage.Define approach for this log entry? looks like this version causes boxing for all 3 params passed.